pkg/asset/installconfig/gcp/validation.go

```diff
@@ -133,3 +134,42 @@ func validateMachineNetworksContainIP(fldPath *field.Path, networks []types.Mach
 	}
 	return field.ErrorList{field.Invalid(fldPath, subnetName, fmt.Sprintf("subnet CIDR range start %s is outside of the specified machine networks", ip))}
 }
+
+//Get all the enabled services for a project and validate if any of the required services are not enabled.
+func ValidateEnabledServices(client API, ctx context.Context, project string) error {
```

Comment:
All exported functions must have a comment that begins with the name of the function, so this comment should be:

```go
// ValidateEnabledServices gets all the enabled services for a project and validates if any of the required services are not enabled.
```

not doing this should cause some of the CI checks to fail.